### ISSUE TITLE
Xcode 9.3 Storyboards, iPhone X, serious break

### ISSUE THREAD

**Issue by justKD** [systers/powerup-iOS#264]
## Description
As a developer,
I need update all storyboard layout constraints,
so that I can ensure the views are properly handled for new constraint system that supports iPhone X.

Currently, opening the storyboard in Xcode 9.3 causes many (maybe most or even all) layout constraints to produce a new warning as Xcode tries to automatically update them. Left as is, it still works fine for older devices.

This seems to be a hefty issue...

## Mocks

<img width="801" alt="screen shot 2018-05-17 at 10 01 08 pm" src="https://user-images.githubusercontent.com/24572031/40214016-207d474a-5a1e-11e8-9b72-84a839b9d6e4.png">

<img width="804" alt="screen shot 2018-05-17 at 10 01 22 pm" src="https://user-images.githubusercontent.com/24572031/40214021-2d8a52c0-5a1e-11e8-91bb-6e56e3a26f81.png">

<img width="800" alt="screen shot 2018-05-17 at 10 01 32 pm" src="https://user-images.githubusercontent.com/24572031/40214027-35b9ff18-5a1e-11e8-8595-276a8976b8cb.png">

<img width="806" alt="screen shot 2018-05-17 at 10 08 10 pm" src="https://user-images.githubusercontent.com/24572031/40214149-d05ed980-5a1e-11e8-8813-fa2de9857a53.png">

### Update [Required]
- [ ] Check and update constraints for all views

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

**Comment by anubhavpulkit** [comment]
Can I work on this issue?

**Comment by sunjunkie** [comment]
@anubhavsingh16  What do you propose to do exactly? This issue involves some UI/graphics work in addition to coding.

**Comment by anubhavpulkit** [comment]
@sunjunkie I want to check and update IBOutlets and IBAction to ensure code is still relevant and functional following storyboard updates.

**Comment by sunjunkie** [comment]
This issue requires UI and graphic updates before IBOutlets and IBActions can be checked or updated. So it cannot be worked on until new graphic elements are created.

**Comment by anubhavpulkit** [comment]
> This issue requires UI and graphic updates before IBOutlets and IBActions can be checked or updated. So it cannot be worked on until new graphic elements are created.

Ok so Can I Work on the update full-screen asses (such as backgrounds) to be aspect ratio agnostic (maybe extra neutral space around the edges)

**Comment by sunjunkie** [comment]
No, that would only be a temporary fix. The full screen assets have to be recreated and the screen layout redone based on the new images. This will require a lot of discussion and UI/image development. This issue is not ready to be worked on until we get the graphics recreated.

**Comment by MANI14011998** [comment]
> This issue requires UI and graphic updates before IBOutlets and IBActions can be checked or updated. So it cannot be worked on until new graphic elements are created.

Hey @sunjunkie I have created new graphic images assets  for all this issue.Is that ok to reopen this issue again

**Comment by justkd** [comment]
This issue is still open. Those closed tags are on other issues that were directed to this particular one. If you have some assets you've made, maybe you could post some examples here, or a link where we can download them, so we can see how they address the issue and discuss what might need to happen to implement a fix?

**Comment by vatsalkul** [comment]
Hey! @sunjunkie @justKD I have made some graphics of buttons used in the app. Please have a look and let me know if I can replace the older Start View Scene Components with new ones. I will add more graphical stuffs in the folder after approval.
[Drive Link](https://drive.google.com/drive/folders/1J-P4hQLq0oqGJI8bCcJRgYLj-A-SYF5f?usp=sharing)

**Comment by MANI14011998** [comment]
![School-3 with chimney](https://user-images.githubusercontent.com/34857581/75185423-134b2e00-576c-11ea-8062-434f2173d616.png)
![WhatsApp Image 2020-02-25 at 1 05 38 AM](https://user-images.githubusercontent.com/34857581/75185430-16461e80-576c-11ea-988b-b5d237f7e6a0.jpeg)
old one
![Screenshot 2020-02-21 at 9 20 52 PM](https://user-images.githubusercontent.com/34857581/75185446-1e9e5980-576c-11ea-8ebc-4db73b3862eb.png)

@justKD have look if this is assign to me i will make whole assets

**Comment by sunjunkie** [comment]
@vatsalkul The images should not have extra pixels. Crop the image as close as possible to the button - as in this screenshot: 
![image](https://user-images.githubusercontent.com/399376/75597568-0881e900-5a4b-11ea-8d65-579d98928ff2.png)

**Comment by sunjunkie** [comment]
Let's break up this issue into smaller pieces. One issue for each screen we want to update. 
@vatsalkul you are welcome to work on the Start View scene.
@MANI14011998 you are welcome to take the Mine Sweeper Game.
@anubhavsingh16  you are welcome to take the map screen.
Each of you please create an issue for just one screen. Once I approve it, you can work on the image assets for that screen. Please follow the issue template.

**Comment by MANI14011998** [comment]
Ohk ma'am i will do that

**Comment by vatsalkul** [comment]
@sunjunkie Please review my [Issue](https://github.com/anitab-org/powerup-iOS/issues/355)

**Comment by anubhavpulkit** [comment]
Ok great

**Comment by anubhavpulkit** [comment]
@sunjunkie I can't find any issue in map screen as mention above I think it is solved so Can I take the Mine Sweeper Game issue?

**Comment by anubhavpulkit** [comment]
There is only one issue on the map which is Smoke of the chimney.

**Comment by anubhavpulkit** [comment]
@sunjunkie So can I work on Minesweeper game issue, please.

**Comment by sunjunkie** [comment]
@anubhavsingh16 There is a big issue with the map - The map should have a blackground image and separate images for the buildings. So there is a work to be done breaking that up.

Anyway, yes, create an issue for the mine sweeper game and I'll review it before you work on it.

**Comment by anubhavpulkit** [comment]
Thank You so much @sunjunkie

**Comment by anubhavpulkit** [comment]
@sunjunkie now only first issue is not solved in this list of issues so can you close this issue and make(or I open) a new one only for the first issue.

**Comment by sunjunkie** [comment]
Ok. I'll take a look.
